app/actions/plugins/index.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {Client4} from '@mm-redux/client';
+import {resetToChannel} from '@actions/navigation';
+
+const RESET_TO_CHANNEL = 'RESET_TO_CHANNEL';
+
+export async function doPluginAction(pluginId, requestURL, body) {
+    try {
+        const response = await Client4.executePluginIntegration(pluginId, requestURL, body);
+        if (response.action === RESET_TO_CHANNEL) {
```

Comment:
The idea behind this is that any action (any time that you click a plugin button and calls the associated requestURL), the plugin can tell you to do "something more" on the client. The idea is to extend eventually this to potentially:
- Open url (browser or natively)
- Go to channel
- Open interactive dialog
- And more...
This was my naïve take on "return to the active channel", but I should just close the menu on click on all menus. This action probably should be removed, since probably is not really useful.

Reply:
It can be useful, but I think we should define the options very clearly and then add them here